src/citrine/resources/user.py

```diff
@@ -46,13 +47,53 @@ def __init__(self,
     def __str__(self):
         return '<User {!r}>'.format(self.screen_name)
 
+    def get(self):
+        """Retrieve a specific user from the database"""
+        raise NotImplementedError("Get Not Implemented in Citrine Platform")
+    
 
-class UserCollection:
+class UserCollection(Collection[User]):
     """Represents the collection of all users."""
 
-    __path_template__ = '/users'
-    _response_key = 'users'
+    _path_template = '/users'
+    _collection_key = 'users'
+    _individual_key = 'user'
     _resource = User
 
     def __init__(self, session: Session):
         self.session: Session = session
+
+    def me(self):
+        """Get information about the current user"""
+        data = self.session.get_resource(f'{self._path_template}/me')
+        return self.build(data)
+
+    def build(self, data):
+        """
+        Build an individual user from a dictionary.
+
+        Parameters
+        ----------
+        data: dict
+          A dictionary representing the user.
+
+        Returns
+        -------
+        User
+          The user created from data.
+
+        """
+        user = User.build(data)
```

Comment:
Is this different than the default implementation of `build` ?

Reply:
Build is abstractmethod in collection so one has to implement it.